cms/static/js/spec/views/course_video_settings_spec.js

```diff
@@ -281,8 +302,286 @@ define(
+                changeProvider('');
+
+                $transcriptProvidersListEl = $courseVideoSettingsEl.find('.transcript-provider-wrapper .transcript-provider-group');
+                expect($transcriptProvidersListEl.find('input[type=radio]').length).toEqual(3);
+                // Verify selected provider view is not shown.
+                expect($courseVideoSettingsEl.find('.transcript-provider-wrapper .selected-transcript-provider')).not.toExist();
+            });
+
+            it('does not show transcript preferences or organization credentials if N/A provider is checked', function() {
+                var $transcriptProvidersListEl;
+
+                renderCourseVideoSettingsView(null, transcriptionPlans);
+
+                // Check N/A provider
+                changeProvider('');
+
+                $transcriptProvidersListEl = $courseVideoSettingsEl.find('.transcript-provider-wrapper .transcript-provider-group');
+                expect($transcriptProvidersListEl.find('input[type=radio]').length).toEqual(3);
+
+                // Verify selected provider view is not shown.
+                expect($courseVideoSettingsEl.find('.transcript-provider-wrapper .selected-transcript-provider')).not.toExist();
+                // Verify transcript preferences are not shown.
+                expect($courseVideoSettingsEl.find('.course-video-transcript-preferances-wrapper:visible')).not.toExist();
+                // Verify org credentials are not shown.
+                expect($courseVideoSettingsEl.find('.organization-credentials-wrapper:visible')).not.toExist();
+            });
+
+            it('shows organization credentials when organization credentials for selected provider are not present', function() {
+                renderCourseVideoSettingsView(null, transcriptionPlans);
+
+                // Check Cielo24 provider
+                changeProvider('Cielo24');
+
+                // Verify organization credentials are shown.
+                expect($courseVideoSettingsEl.find('.organization-credentials-wrapper:visible')).toExist();
+
+                // Verify transcript preferences are not shown.
+                expect($courseVideoSettingsEl.find('.course-video-transcript-preferances-wrapper:visible')).not.toExist();
+            });
+
+            it('shows transcript preferences when organization credentials for selected provider are present', function() {
+                renderCourseVideoSettingsView(null, transcriptionPlans, transcriptOrganizationCredentials);
+
+                // Check Cielo24 provider
+                changeProvider('Cielo24');
+
+                // Verify organization credentials are not shown.
+                expect($courseVideoSettingsEl.find('.organization-credentials-wrapper:visible')).not.toExist();
+
+                // Verify transcript preferences are shown.
+                expect($courseVideoSettingsEl.find('.course-video-transcript-preferances-wrapper:visible')).toExist();
```

Comment:
We should write a util that checks if the correct view is rendered with expected fields. I think only `toExist` is not enough.